libpod/image/image.go

```diff
@@ -1289,58 +1279,11 @@ func areParentAndChild(parent, child *imgspecv1.Image) bool {
 
 // GetParent returns the image ID of the parent. Return nil if a parent is not found.
 func (i *Image) GetParent(ctx context.Context) (*Image, error) {
-	var childLayer *storage.Layer
-	images, err := i.imageruntime.GetImages()
+	tree, err := i.imageruntime.layerTree()
```

Comment:
I’d prefer the caller to have to supply the `layerTree` to encourage caching it over multiple calls.